.github/workflows/build-and-push-image.yml

```diff
@@ -45,25 +60,24 @@ jobs:
           context: .
           file: Dockerfile.gpaas-azure-migration
           tags: |
-            ${{ secrets.CIP_ACR_URL }}/${{ env.DOCKER_IMAGE }}:${{ env.DOCKER_IMAGE_TAG }}
-            ${{ secrets.CIP_ACR_URL }}/${{ env.DOCKER_IMAGE }}:${{ env.BRANCH_TAG }}
+            ${{ secrets.CIP_ACR_URL }}/${{ env.DOCKER_IMAGE }}:${{ needs.set-env.outputs.branch }}
```

Comment:
What is needs.set-env.outputs*?

Reply:
The `build` job relies on a previous job called `set-env` which sets three outputs.

**outputs:**
```
environment
branch
release
```
**example values:**
```
dev
main
dev-2023-01-13.5
```